mu_repo/__init__.py

```diff
@@ -66,35 +66,39 @@ def CreateConfig(config_file='.mu_repo'):
 
     return Config.Create(contents)
 
+
 def CreateParams(args, config_file='.mu_repo'):
     return Params(CreateConfig(config_file), args, config_file)
 
 
-def SearchConfigFile(start_dir, recurse_limit=20):
+def SearchConfigFile(start_dir, name='.mu_repo', recurse_limit=20):
     """Search upwards in the file system starting in start_dir for a ".mu_repo" file, until the maximum limit or
     the root of the file-system is reached.
     """
-    start_dir = os.path.normpath(os.path.abspath(start_dir))
-    while True:
-        filename = os.path.join(start_dir, '.mu_repo')
+    start_dir = os.path.abspath(start_dir)
+    while recurse_limit >= 0:
+        filename = os.path.join(start_dir, name)
         if os.path.isfile(filename):
             return filename
 
-        recurse_limit -= 1
-        if recurse_limit == 0:
+        # give up searching if we are in a .git repository
+        if os.path.isdir(os.path.join(start_dir, '.git')):
             return None
```

Comment:
Sure, good idea. I had forgot about the use case of using mu just for git commands. 

Done!